### PR TITLE
Expose normalization method to Mel transforms

### PR DESCRIPTION
**Pull request by jieruan02**
This PR attempts to address issue #1207

### CODE REVIEW

**Review by mthrok — APPROVED** [review]
Looks good. Thanks!

File: torchaudio/transforms.py [hunk 1]
```diff
@@ -418,6 +426,8 @@ class MelSpectrogram(torch.nn.Module):
             :attr:`center` is ``True``. Default: ``"reflect"``
         onesided (bool, optional): controls whether to return half of results to
             avoid redundancy. Default: ``True``
+        norm (Optional[str]): If 'slaney', divide the triangular mel weights by the width of the mel band
+        (area normalization). (Default: ``None``)
```
Comment (rbracco) [inline torchaudio/transforms.py:430]: Hi, thank you for this PR. This line has a small typo that is causing a formatting issue in the docs. 
image

Line 430 should be indented so the docs generator doesn't think it's a new argument. Thank you!
